### DISCUSSION TITLE
how to use string_of_metadata to change title of song?

### DISCUSSION THREAD

**Discussion by ttodua** [savonet/liquidsoap#4112] [Q&A] [answered]
https://www.liquidsoap.info/doc-dev/reference.html#string_of_metadata

**Comment by vitoyucepi** [comment, marked as answer]
Hi @ttodua,
You should use `map_metadata` to change the title of the song.
Here's an article about it https://www.liquidsoap.info/doc-2.2.5/metadata.html.
